sdk/storage_datalake/src/operations/file_system_delete.rs

```diff
@@ -0,0 +1,86 @@
+use crate::clients::FileSystemClient;
+use azure_core::prelude::*;
+use azure_core::{
+    collect_pinned_stream,
+    headers::{add_mandatory_header2, add_optional_header2},
+    AppendToUrlQuery, Response as HttpResponse,
+};
+use azure_storage::core::headers::CommonStorageResponseHeaders;
+use std::convert::TryInto;
+
+/// A future of a create file system response
+type DeleteFileSystem =
+    futures::future::BoxFuture<'static, crate::Result<DeleteFileSystemResponse>>;
+
+#[derive(Debug, Clone)]
+pub struct DeleteFileSystemBuilder {
+    client: FileSystemClient,
+    if_modified_since_condition: Option<IfModifiedSinceCondition>,
+    client_request_id: Option<ClientRequestId>,
+    timeout: Option<Timeout>,
+}
+
+impl DeleteFileSystemBuilder {
+    pub(crate) fn new(client: FileSystemClient) -> Self {
+        Self {
+            client,
+            if_modified_since_condition: None,
+            client_request_id: None,
+            timeout: None,
+        }
+    }
+
+    setters! {
+        if_modified_since_condition: IfModifiedSinceCondition => Some(if_modified_since_condition),
+        client_request_id: ClientRequestId => Some(client_request_id),
+        timeout: Timeout => Some(timeout),
+    }
+
+    pub fn into_future(self) -> DeleteFileSystem {
+        let this = self.clone();
+        let ctx = self.client.context.clone();
+
+        Box::pin(async move {
+            let mut url = this.client.url().clone();
+            self.timeout.append_to_url_query(&mut url);
+            url.query_pairs_mut().append_pair("resource", "filesystem");
+
+            debug!("url = {}", url);
```

Comment:
Remove this too - it's not descriptive enough and will make debug logging very noisy.

Reply:
gone :)